app/controllers/stations_controller.rb

```diff
@@ -154,13 +154,15 @@ def embed
   # Used by Ardiuno to lookup station ID
   def find
     @station = Station.find_by(hw_id: params[:hw_id])
-    if(@station.nil?)
+    if @station.nil?
       respond_to do |format|
         format.json { head :not_found }
       end
     else
+      # Update station to show that it has been intialized
+      @station.unresponsive! if @station.not_initialized?
```

Comment:
Hmm, can be removed then since creating observations changes the status to active.